src/reducers/usersReducer.js

```diff
@@ -16,9 +16,11 @@ export default function users(state = initialState.Users, {payload, type}) {
         case actions.PATCH_ACCOUNT_NOTE_SUCCESSFUL:
             return handleAccountNotesPost(state, payload);
         case actions.POST_STUDENT_SUCCESSFUL:
-            return handleStudentPost(state,payload);
+            return handleStudentPost(state, payload);
         case actions.GET_ACCOUNT_SEARCH_QUERY_SUCCESS:
-            return handleAccountSearchResults(state,payload);
+            return handleAccountSearchResults(state, payload);
+        case actions.FETCH_OOO_SUCCESS:
```

Comment:
support POST_OOO_SUCCESS - otherwise, when you post an OOO, you don't see it immediately reflected on the instructor's schedule